fastlane/Fastfile

```diff
@@ -93,7 +93,7 @@ platform :ios do
   desc "Deletes the archive generated by gym."
   lane :cleanupArchive do
     if "#{lane_context[SharedValues::XCODEBUILD_ARCHIVE]}" != ""
-      sh "rm -r \"#{File.dirname(lane_context[SharedValues::XCODEBUILD_ARCHIVE])}\""
+      sh "rm -r \"#{File.dirname(lane_context[SharedValues::XCODEBUILD_ARCHIVE])}\" || :"
```

Comment:
What's this `|| :` doing?

Reply:
Ignores the output in case it fails (e.g. projects using non-standard build directories).

Reply:
👍